lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -117,15 +117,45 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
-    case 'd2l_file_not_found_in_course':
+    case 'd2l_file_not_found_in_course_instructor':
```

Comment:
Different code for different role. If we start using this pattern more we might want to look into adding more metadata to the error response (role, product...)

For now just a different error code is simple enought.